auth/src/PhoneAuthProvider.cs

```diff
@@ -57,13 +57,6 @@ namespace Firebase.Auth {
 ///       send an SMS. Credential is automatically created and passed to
 ///       the app via @ref VerificationCompleted.
 public sealed class PhoneAuthProvider : global::System.IDisposable {
-  /// Maximum value of `autoVerifyTimeOutMs` in @ref VerifyPhoneNumber.
-  /// @ref VerifyPhoneNumber will automatically clamp values to this amount.
-  public static uint MaxTimeoutMs {
-    get {
-      return PhoneAuthProviderInternal.kMaxTimeoutMs;
-    }
-  }
```

Comment:
As mentioned in the C++ SDK, it's better to deprecate this field instead of removing it as this is a breaking change.

Reply:
Alright. Added them back. Used both `System.Obsolete` and `@deprecated` for both compiler warning and ref doc